app/benchmarking.py

```diff
@@ -56,54 +61,182 @@ def run(circuit, backend, token, shots, benchmark_id, original_depth, original_w
     return content_location
 
 
-def randomize(qpu_name, num_of_qubits, shots, min_depth_of_circuit, max_depth_of_circuit, num_of_circuits, token):
+def calc_wd(qpu_name):
+    """calculates the wd-value of a Quantum Computer based on the clifford data in your database"""
+    wd = []
+    wd_count = np.zeros([7, 5])
+    min_sample_size = 15
+    wd_success_count = np.zeros([7, 5])
+    min_histogram_intersection = 0.75
```

Comment:
mark them somehow with comments such that it is clear that this is a adaptable threshold